### DISCUSSION TITLE
Calls to /api/verify from ip 3.85.226.144 (aws)?

### DISCUSSION THREAD

**Discussion by lakemike** [photoprism/photoprism#1459] [Q&A] [answered]
Hi, I'm running photoprism behind a combination of traefik and authelia (2FA). I see some error messages in authelia's docker logs:
`error: date='Thu Aug 05 2021 10:20:17 GMT+0000 (UTC)' method='GET', path='/api/verify' requestId='a74d27c6-0bb2-47ae-b631-b7b189b0cf56' sessionId='ojx4qDN-w1nSkhwS30i_dna5exDGo7oG' ip='3.85.226.144' message='Error: User 'unknown' is not sufficiently authorized to access photoprism.mydomain.xx/.'`
Reverse DNS says 3.85.226.144 is an AWS server, so somehow photoprism is triggering calls from AWS to my photoprism container. What are these calls to /api/verify and do I need to enable them?
Thanks!

**Comment by lastzero** [comment, marked as answer]
All our API calls use `/api/v1/...`, so the `/api/verify` request doesn't seem to come from our client or server. Consider using a VPN if it's a private instance anyway as putting multiple layers of auth in top of PhotoPrism is hard to configure and may take time to experiment e.g. because it impacts browser caching and we're using websockets.
